site/docs/design_tokens/typography.mdx

```diff
@@ -37,47 +35,92 @@ Typography can be used to organise information and help users find their way aro
 
 
 ## Usage
-### Headings
+The HDS offers two different sets of font size tokens for heading and body text. The tokens define font size in rem values.
+
+### Heading sizes
+The heading font size tokens are heading level agnostic. This allows you to fit the **heading scale** to the specific needs of the service. In addition to predefined _default_ and _small_ heading scale, custom scales can be also used.
+
 <Playground>
-  <h1>Heading H1 (3.25 rem)</h1>
-  <h2>Heading H2 (2.25 rem)</h2>
-  <h3>Heading H3 (1.5 rem)</h3>
-  <h4>Heading H4 (1.25 rem)</h4>
-  <h5>Heading H5 (1.125 rem)</h5>
-  <div className="subtitle">Subtitle (1 rem)</div>
+  <h1 style={{ fontSize: 'var(--fontsize-heading-xl) '}}>Heading XL (52 pt)</h1>
```

Comment:
I think we should use pixel values, `pt` is mostly for print afaik.

Reply:
Fixed pt values to px values in typography token documentation tables.

It's a common debate wether to use pt or px, but after discussion with HDS team, we decided that px is more suitable in HDS context.